refinedstorage2-platform-common/src/main/java/com/refinedmods/refinedstorage2/platform/common/screen/AbstractBaseScreen.java

```diff
@@ -217,4 +218,9 @@ protected boolean tryOpenResourceFilterAmountScreen(final Slot slot) {
         }
         return canChangeAmount;
     }
+
+    @Nullable
+    public Slot getHoveredSlot() {
+        return this.hoveredSlot;
+    }
```

Comment:
This is exposing Too Much Information.

```suggestion
    @Nullable
    public FilteredResource<?> getFilteredResource() {
        return hoveredSlot instanceof ResourceFilterSlot resourceFilterSlot
            ? resourceFilterSlot.getFilteredResource()
            : null;
    }
```